tests/storage/hostdisk.go

```diff
@@ -0,0 +1,27 @@
+package storage
+
+import (
+	"fmt"
+	"path/filepath"
+
+	k8sv1 "k8s.io/api/core/v1"
+
+	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/tests/framework/checks"
+	"kubevirt.io/kubevirt/tests/libpod"
+)
+
+func CreateHostDiskImage(diskPath string) *k8sv1.Pod {
```

Comment:
Nit: Can we take this opportunity to name this properly? `CreateDiskonHost?`

Reply:
+1 for a separate commit with a clear motivation.

Reply:
added